homeassistant/components/london_underground/sensor.py

```diff
@@ -53,16 +58,48 @@ async def async_setup_platform(
     )
```

Comment:
And yes, we shoud import the old config

Reply:
done in https://github.com/home-assistant/core/pull/152050/commits/1a219352e85b6638a2333aec332863634cb83796